### PR TITLE
Remove password attribute from connections object before logging

### PR DESCRIPTION
**Pull request by SafalPandey**
* Remove password attribute from connections object before logging

### CODE REVIEW

File: src/util/object.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/**
+ * Get the copy of object without attributes.
+ *
+ * @param {any} obj
+ * @param {any[]} attrsToExclude
+ * @returns {T}
+ */
+export function withoutAttrs<T>(obj: any, attrsToExclude: any[]): T {
```
Comment (mesaugat) [inline src/util/object.ts]: This is an overkill at the moment.

Comment (mesaugat) [reply src/util/object.ts]: Let's remove this util.

File: src/config.ts [hunk 2]
```diff
@@ -52,7 +53,7 @@ export async function resolveConnections(): Promise<Connection[]> {
     id: connection.id || `${connection.host}/${connection.database}`
   }));
 
-  log('Resolved connections: %O', connections);
+  log('Resolved connections: %O', listWithoutAttrs<Connection>(connections, ['password']));
```
Comment (mesaugat) [inline src/config.ts]: Let's map through the connections and only output the database name.

File: src/config.ts [hunk 3]
```diff
@@ -43,7 +44,7 @@ export async function resolveConnections(): Promise<Connection[]> {
   const loaded = await fs.read(filename);
   const { connections } = JSON.parse(loaded);
 
-  log('Connections parsed: %o', connections);
+  log('Connections parsed: %o', listWithoutAttrs<Connection>(connections, ['password']));
```
Comment (mesaugat) [inline src/config.ts]: No need to log anything here.

File: src/util/object.ts (hunk 1 above)
Comment (mesaugat) [reply src/util/object.ts]: Also, if we are to add such utils we will need tests for it.

File: src/config.ts [hunk 4]
```diff
@@ -43,16 +43,21 @@ export async function resolveConnections(): Promise<Connection[]> {
...
-
   // TODO: Validate the connections received from file.
 
   const result = connections.map((connection: Connection) => ({
     ...connection,
     id: connection.id || `${connection.host}/${connection.database}`
   }));
 
-  log('Resolved connections: %O', connections);
+  log('Resolved connections: %O', connections.map((connection: Connection) => {
```
Comment (kabirbaidhya) [inline src/config.ts]:
```ts
({ id, host, database }) => ({ id, host, database })
```

**Pull request merged**